app/controllers/ClientController.scala

```
@@ -69,9 +75,40 @@ class ClientController @Inject()(appConfig: AppConfig,
         Future.successful(Ok(views.html.individual.correspondenceDetails(appConfig, correspondenceDetailsForm.correspondenceDetailsForm)))
   }
 
-  val submitIndividualCorrespondenceDetails = TODO
+  val submitIndividualCorrespondenceDetails: Action[AnyContent] = authorisedActions.authorisedAgentAction {
+    implicit user =>
+      implicit request =>
+
+        def createRelationship(account: AgentAccount, reference: SubscriptionReference) = {
+          relationshipService.createClientRelationship(RelationshipModel(account.agentCode.value, reference.cgtRef)).flatMap {
+            case SuccessfulRelationshipResponse => Future.successful(Redirect(routes.ClientController.confirmation(reference.cgtRef)))
+            case _ =>
+              Logger.warn("Agent Client relationship creation failed.")
+              Future.successful(InternalServerError)
+          }
+        }
+
+        def successAction(model: CorrespondenceDetailsModel): Future[Result] = {
+          lazy val arnAccount = user.authContext.principal.accounts.agent
+
+          clientService.subscribeIndividualClient(model).flatMap {
+            reference =>
+              arnAccount match {
+                case Some(account) => createRelationship(account, reference)
+                case None =>
+                  Logger.warn("No ARN was supplied")
+                  Future.successful(InternalServerError)
```

Do not return a Future.successful(InternalServerError), let the method fail so the automated error page is returned.